model/src/test/kotlin/utils/DependencyGraphConverterTest.kt

```diff
@@ -135,9 +157,17 @@ private fun createIdentifier(managerName: String, index: Int, forProject: Boolea
  */
 private fun createDependencies(managerName: String, startIndex: Int, count: Int): SortedSet<PackageReference> =
     (startIndex..(startIndex + count)).mapTo(sortedSetOf()) { index ->
-        PackageReference(createIdentifier(managerName, index, forProject = false))
+        PackageReference(createIdentifier(managerName, index, forProject = false), issues = createIssues(index))
     }
 
+/**
+ * Create a list with issues for a test dependency based on its [index].
+ */
+private fun createIssues(index: Int): List<OrtIssue> =
+    emptyList<OrtIssue>().takeIf { index % 2 == 0 } ?: listOf(
+        OrtIssue(source = "test", message = "Test issue $index.")
+    )
```

Comment:
Alternatively:

    listOfNotNull(
        OrtIssue(source = "test", message = "Test issue $index.").takeUnless { index % 2 == 0 }
    )

Reply:
IMHO the original approach is more efficient as an `OrtIssue` object is only created if necessary. Regarding the readability, I see no major difference.